Core/HLE/sceUtility.cpp

```diff
@@ -293,7 +293,11 @@ static u32 sceUtilityLoadModule(u32 module) {
 	u32 allocSize = info->size;
 	char name[64];
 	snprintf(name, sizeof(name), "UtilityModule/%x", module);
-	currentlyLoadedModules[module] = userMemory.Alloc(allocSize, false, name);
+	if (allocSize != 0) {
```

Comment:
You're a good fan of those, aren't you?

In this case, `userMemory.Alloc()` has side effects - it's not just an accessor, it does stuff.  I usually avoid ternaries in these cases.  Just like here:

```c++
Something somethings = c != 0 ? new Something[c] : nullptr;
```

Would seem weird, even if it's not very long.  The other reason though is that this line is already kinda long, I didn't want to complicate it further.

Ternaries are great in some cases.  Here are some examples I like:
https://github.com/unknownbrackets/ppsspp/blob/hle-minor/Core/HLE/sceKernelThread.cpp#L743
https://github.com/unknownbrackets/ppsspp/blob/hle-minor/Core/HLE/sceKernelThread.cpp#L1414
https://github.com/unknownbrackets/ppsspp/blob/hle-minor/Core/HLE/sceKernelThread.cpp#L1960
https://github.com/unknownbrackets/ppsspp/blob/hle-minor/Core/HLE/sceKernelThread.cpp#L2923
https://github.com/unknownbrackets/ppsspp/blob/hle-minor/GPU/GLES/Framebuffer.cpp#L379
https://github.com/unknownbrackets/ppsspp/blob/hle-minor/GPU/GLES/Framebuffer.cpp#L536

In general, I'm not trying to keep SLOC low and create this:
```c
// Actual C program.
main(v,c)char**c;{for(v[c++]="Hello, world!\n)";(!!c)[*c]&&(v--||--c&&execlp(*c,*c,c[!!c]+!!c,!c));**c=!c)write(!!*c,*c,!!**c);}
```

I am trying to maximize code readability.  There's a balance between use of ternaries and... that, right above.

-[Unknown]